recipe/meta.yaml

```diff
@@ -6,43 +6,113 @@ package:
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_constraints.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_data_types.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_debug.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_empty.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_ephemeral.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_grants.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_incremental.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_incremental_microbatch.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_persist_docs.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_query_comment.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_relations.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_show.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_simple_copy.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_simple_snapshot.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_store_test_failures.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_unit_testing.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_utils.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_simple_seed/test_simple_seed.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_hooks/test_hooks.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/materializations/materialized_view_tests/test_materialized_view.py" %}
+
+{% set passed_tests = "" %}
+{% set passed_tests = " tests/functional/test_config.py::TestProfile::test_bad_type" %}
+{% set passed_tests = passed_tests + " tests/functional/test_config.py::TestProfile::test_from_raw_profiles" %}
+{% set passed_tests = passed_tests + " tests/functional/test_config.py::TestProfile::test_invalid_credentials" %}
+{% set passed_tests = passed_tests + " tests/functional/test_config.py::TestProfile::test_missing_target" %}
+{% set passed_tests = passed_tests + " tests/functional/test_config.py::TestProfile::test_missing_type" %}
+{% set passed_tests = passed_tests + " tests/functional/test_connection_manager.py::TestConnectionManagerOpen::test_open" %}
+
 test:
+  source_files:
+    - tests
   imports:
     - dbt.adapters.postgres
   commands:
     - pip check
+    - python -m pytest -vvv tests/unit  [not win]
+    - cmd.exe /C "python -m pytest -vvv tests/unit"  [win]
```

Comment:
Do we need the `cmd.exe /C"..."` wrapper?  (Ditto the one below.)

Reply:
Windows gha fails with:
```
'tests' is not recognized as an internal or external command,
operable program or batch file.
```

Reply:
Is that because on Windows the path separator is `\` whereas `/` is the start of an option?

```
python -m pytest -vvv tests\unit
```